include/netlink/utils.h

```diff
@@ -231,6 +231,13 @@ enum {
 	NL_CAPABILITY_NL_ADDR_FILL_SOCKADDR = 21,
 #define NL_CAPABILITY_NL_ADDR_FILL_SOCKADDR NL_CAPABILITY_NL_ADDR_FILL_SOCKADDR
 
+	/**
+	 * Support omitting @ctx_str argument to xfrmnl_sa_get_sec_ctx() to check
+	 * for required buffer size for context string.
+	 */
+	NL_CAPABILITY_SEC_CTX_LEN = 22,
```

Comment:
Makes sense. However, since the SA_KEY_SIZE macro is has a NL_CAPIBILITY_XFRM_ prefix -- without the NL, I called the macro  NL_CAPABILITY_XFRM_SEC_CTX_LEN. Looks better to me. But feel free to change anything you don't like.